highcharts_core/js_literal_functions.py

```diff
@@ -396,7 +396,7 @@ def get_js_literal(item, careful_validation = False) -> str:
             as_str += f"""'{item}'"""
         else:
             as_str += f"""{item}"""
-    elif item == constants.EnforcedNull:
+    elif (item == constants.EnforcedNull) or np.isnan(item):
```

Comment:
This is a good catch, and a reasonably good solution. However, NumPy is a soft dependency for Highcharts for Python. This means that the function still has to work even if someone doesn't have NumPy installed / is not using NumPy. The `js_literal_functions.py` file has a useful Boolean variable `HAS_NUMPY` which is `True` if NumPy is available and `False` if not. I'd recommend breaking this check out into a separate `elif` clause along the lines of:

```
elif HAS_NUMPY and np.isnan(item):
    as_str += """null"""
```

With this adjustment, it should work properly to fix #200 and still work well for users who are not using / do not have NumPy available in their runtime environments.

Reply:
Thank you for the quick reply and the suggestion. I will take a look later today and assert it doesn't yield errors for someone without numpy installed as well. If it keeps the functionallity I will make the change for the PR. Let me know when you make the merge! 😎